UT4MasterServer.Services/Scoped/RatingsService.cs

```diff
@@ -118,21 +120,35 @@ public async Task<PagedResponse<RankingsResponse>> GetRankingsAsync(string ratin
 
 		var accountIds = ratings.Select(s => s.AccountID);
 		var filterAccounts = Builders<Account>.Filter.In(f => f.ID, accountIds);
-		var fields = Builders<Account>.Projection.Include(i => i.Username);
-		var userNames = await accountsCollection
+		var accounts = await accountsCollection
 			.Find(filterAccounts)
-			.Project(p => new { p.ID, p.Username })
+			.Project(p => new { p.ID, p.Username, p.CountryFlag })
 			.ToListAsync();
 
 		var rank = skip;
 		var rankings = ratings
-			.Select(s => new RankingsResponse()
+			.Select(s =>
 			{
-				Rank = ++rank,
-				AccountID = s.AccountID,
-				Player = userNames.FirstOrDefault(f => f.ID == s.AccountID)?.Username ?? UnknownUser,
-				Rating = s.RatingValue / Rating.Precision,
-				GamesPlayed = s.GamesPlayed,
+				var userName = UnknownUser;
+				var countryFlag = DefaultCountryFlag;
+				if (accounts.FirstOrDefault(f => f.ID == s.AccountID) is { } account)
+				{
+					userName = account.Username;
+					if (account.CountryFlag is not null)
+					{
+						countryFlag = account.CountryFlag.Replace('.', ' ');
```

Comment:
The way that flags are stored is frontend's problem. Changing this filename depends on the idea that frontend has flags stored with spaces instead of dots in filenames. Frontend has to deal with this string transformations.

Reply:
So, should I change all spaces to dots on frontend in file names?

Reply:
Replaced on FE.